### PR TITLE
Feature/unit test

### PR DESCRIPTION
**Pull request by praslnx8**

### CODE REVIEW

File: app/src/main/java/com/mymeetings/android/services/LocalCalendarChangeEventReceiver.kt [hunk 1]
```diff
@@ -9,13 +9,14 @@ import com.mymeetings.android.model.strategies.CalendarFetchStrategyType
...
-class LocalCalendarChangeEventReceiver()
+class LocalCalendarChangeEventReceiver
     : BroadcastReceiver(), KoinComponent {
 
     private val calendarEventsSyncManager: CalendarEventsSyncManager = get()
+    private val consoleLog : ConsoleLog = get()
 
     override fun onReceive(context: Context?, intent: Intent?) {
-        ConsoleLog.i(message = "Receiver received")
+        consoleLog.i(message = "on received called")
```
Comment (amitd291) [inline app/src/main/java/com/mymeetings/android/services/LocalCalendarChangeEventReceiver.kt]: Should the log message be _`onReceive called`_?

File: app/src/test/java/com/mymeetings/android/model/extensions/MapperExtensionsKtTest.kt [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        val expectedData = CalendarEventsDbModel(
+            id = id,
+            title = title,
+            startTime = startTime,
+            endTime = endTime,
+            isDone = isDeleted
+        )
+
+        assertEquals(expectedData, calendarEvent.toDbModel())
```
Comment (amitd291) [inline app/src/test/java/com/mymeetings/android/model/extensions/MapperExtensionsKtTest.kt]: Can we try `Truth` assertions?

File: app/src/test/java/com/mymeetings/android/utils/ClockUtilsTest.kt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        val expected = System.currentTimeMillis()
+
+        assertTrue(expected - actual < delta)
+    }
+
+    @Test
+    fun `getFormattedTime should format time as hh mm a pattern`() {
+        val timeInMillis = 1579530080000 //time of 07:51 for GMT +5.30
+
+        assertEquals("07:51 PM", ClockUtils().getFormattedTime(timeInMillis))
```
Comment (amitd291) [inline app/src/test/java/com/mymeetings/android/utils/ClockUtilsTest.kt]: Can we try `Truth` assertions?

File: app/src/main/java/com/mymeetings/android/services/LocalCalendarChangeEventReceiver.kt (hunk 1 above)
Comment (praslnx8) [reply app/src/main/java/com/mymeetings/android/services/LocalCalendarChangeEventReceiver.kt]: Changed

File: app/src/test/java/com/mymeetings/android/model/extensions/MapperExtensionsKtTest.kt (hunk 2 above)
Comment (praslnx8) [reply app/src/test/java/com/mymeetings/android/model/extensions/MapperExtensionsKtTest.kt]: Added Truth

File: app/src/test/java/com/mymeetings/android/utils/ClockUtilsTest.kt (hunk 3 above)
Comment (praslnx8) [reply app/src/test/java/com/mymeetings/android/utils/ClockUtilsTest.kt]: Added truth

**Pull request merged**